submissions/Semka094/frogger/app.js

```diff
@@ -0,0 +1,127 @@
+const gameLevel = document.createElement('div');
+gameLevel.textContent = 'Your level: 0';
+
+document.body.append(gameLevel);
+
+const xStep = 100;
+const yStep = 85;
+
+const coordinates = {
```

Comment:
See examples of constants. By convention constants are ALL_CAPS. This way we know it is a constant when we come across it in the code. It is sufficient to make "root" constants ALL_CAPS, i.e. no need capitalizing properties of "container" constants.

Reply:
I found an article where the author stated that [it is not obligatory](https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/) to name constants in JavaScript in the capital. 
Also, I checked [Airbnb guidance on const naming](https://github.com/airbnb/javascript/#naming--uppercase) where they told that:
> You may optionally uppercase a constant only if it (1) is exported, (2) is a const (it can not be reassigned), and (3) the programmer can trust it (and its nested properties) to never change.

Could you share any reference where it is stated that by convention constants in JS should be ALL_CAPS?

Reply:
These are all (just) conventions. The linked requirements are closer to the project than airbnb's or google's codestyle guides.

You may skip this. It will just take a reviewer longer to distinguish between global variables and constants.